test/instrumentation/modules/http2.test.js

```diff
@@ -602,8 +602,6 @@ function assertPath(t, trans, secure, port, path, httpVersion) {
   if (trans.context.request.headers.traceparent) {
     expectedReqHeaders.traceparent = trans.context.request.headers.traceparent;
     expectedReqHeaders.tracestate = trans.context.request.headers.tracestate;
-    expectedReqHeaders['elastic-apm-traceparent'] =
```

Comment:
Note for reviewer: http2 tests were assuming `useElasticTraceparent` default to `true`. I wonder if we have to refactor it so we tests both cases.

Reply:
I am fine not bothering with a test of `useElasticTraceparentHeader: true` for http2. We have one for http tests and that's good enough.